src/modules/v1/evidence/index.ts

```
@@ -29,14 +37,102 @@ export function EvidenceModule(
             reply.send({ hello: 'world' });
         }
     );
+    
+    fastify.post('/upload', async (request, reply) => {
+        const data = await request.file();
+        if(!data) {
+            return reply.send({ message: 'No file uploaded' });
```

Let's fail fast, if no fail let's reply with `400`